fw/http_stream.h

```diff
@@ -41,8 +41,6 @@ typedef enum {
 	HTTP2_STREAM_LOC_RESERVED,
```

Comment:
Shouldn't we fix the comment above in reference to a newer RFC and don't the reduced set of states now is fully compliant with the RFC set?

Reply:
No we still have HTTP2_STREAM_LOC_CLOSED and HTTP2_STREAM_REM_CLOSED states, which are not described in the RFC.